withDom.js

```diff
@@ -1,3 +1,5 @@
+require('raf/polyfill');
```

Comment:
i'm not sure. this could cause unexpected behavior elsewhere in someone's codebase. That said, so could adding this polyfill... but without it, *nothing* works.... so we kinda need it.